pep/app/pods/read/document/template.hbs

```diff
@@ -33,36 +33,79 @@
                 <p class="small text-muted {{if this.model.accessLimited " mb-2" " mb-0"}}">
                     {{this.model.documentRef}}
                 </p>
-                {{#unless this.model.accessLimited}}
-                    <div class="flex-shrink-0">
-                        <DropdownMenu @horizontalPosition="auto-right" as |DD|>
-                            <DD.Trigger @htmlTag="button" class="btn btn-link px-2">
-                                <FaIcon @icon="download" @size="lg" @prefix="fas" />
-                            </DD.Trigger>
-                            <DD.Content class="dropdown-menu">
-                                <DropdownAction @dd={{DD}} @action={{fn this.downloadDocument this.downloadUrlEpub}}>
-                                    {{t "search.preview.download.ePub"}}
-                                </DropdownAction>
-                                <DropdownAction @dd={{DD}} @action={{fn this.downloadDocument this.downloadUrlPdf}}>
-                                    {{t "search.preview.download.pdf"}}
-                                </DropdownAction>
-                                <DropdownAction @dd={{DD}} @action={{fn this.downloadDocument this.downloadUrlPdfOrig}}>
-                                    {{t "search.preview.download.pdfOrig"}}
-                                </DropdownAction>
-                            </DD.Content>
-                        </DropdownMenu>
+                <div class="d-flex align-items-center">
+                    <span class="p-1 d-flex align-items-center">
+                        {{#if this.showPreviousSearchHitButton}}
+                            <Button
+                                @action={{this.viewPreviousSearchHit}}
+                                @type="link"
+                                @icon="chevron-left"
+                                @iconPrefix="fas"
+                                @iconSize="lg"
+                                @tooltip={{t "common.previous"}}
+                                class="py-0 px-1 mx-1"
+                            />
+                        {{/if}}
+                        {{this.q}}
```

Comment:
do search hits only come from "smart search"-based queries (which is what `q` holds), or can hits also occur from fulltext search term-based hits? If its the latter, i think you'd need a bit more logic here to build this displayed text value